tests/foreman/ui/test_myaccount.py

```diff
@@ -16,96 +16,205 @@
 @Upstream: No
 """
 
+from fauxfactory import gen_string, gen_alpha
+from nailgun.entities import User
+from robottelo.constants import LANGUAGES
+from robottelo.datafactory import generate_strings_list
 from robottelo.decorators import stubbed, tier1
 from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators
+from robottelo.ui.locators import locators
+from robottelo.ui.session import Session
+
+
+def _valid_string_data(min_len=1, max_len=50):
+    """Generates valid string data for ui tests
+
+    :param min_len: min len for generated string
+    :param max_len: max len for generated string. Default is 50 because of
+    ui validation
+    :return: list of generated strings
+    """
+    return generate_strings_list(
+        min_length=min_len, max_length=max_len, exclude_types=['html']
+    )
 
 
 class MyAccountTestCase(UITestCase):
     """Implements Users tests in UI
 
     [1] Positive Name variations - Alpha, Numeric, Alphanumeric, Symbols,
-    Latin1, Multibyte, Max length,  Min length, Max_db_size, html, css,
+    Latin1, Multibyte, Max length,  Min length, Max_db_size, css,
     javascript, url, shell commands, sql, spaces in name
 
-    [2] Negative Name Variations -  Blank, Greater than Max Length,
+    [2] Negative Name Variations -  html, Blank, Greater than Max Length,
     Lesser than Min Length, Greater than Max DB size
 
     """
 
-    @stubbed()
+    def setUp(self):
+        """Setup of myaccount"""
+        super(MyAccountTestCase, self).setUp()
+        # Creating user for each test to not mess with default user
+        user = User(login=gen_alpha(), password='password')
+        # copying missing id because create returns only password's hash once
+        # it is not stored in plain text for security reason
+        user.id = user.create().id
```

Comment:
not sure that I get the message here. Yes, password is not returned, but id is, so why do you perform that operation?